src/client/wallet/ClaimRewardsBlock.js

```diff
@@ -55,7 +55,7 @@ class ClaimRewardsBlock extends Component {
           loading: false,
           rewardClaimed: true,
         });
-        this.props.getUserAccountHistory(name).then(() => this.props.updateAuthUser(name));
+        this.props.getUserAccountHistory(name).then(() => this.props.reload());
```

Comment:
I'm not sure why would that happen.
The only difference now is that `loadingUpdateAuthUser` is never changed (I wasn't used anyway), and metadata is now a string instead of an object (that was causing the bug).
I'm pretty sure it was working when I tried, can't try again as I already claimed rewards.